index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/7bfe064e41.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio Page</title>
+</head>
+<body>
+ <header>
+ <h1><a href="#">Atril</a></h1>
+ <div class="bars">
+    <div class="bar"></div>
+    <div class="bar"></div>
+    <div class="bar"></div>
+ </div>
+ </header>
+<div class="content-items">
+<img src="Images/background_img.png" alt="Background Image">
+<div class="header-text">
+    <h2>I’m Atril <br>
+        Glad to see you!</h2>
+</div>
+<div class="paragraph-text">
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+</div>
+<div class="contact_list">
+    <h3><a href="#">LET’S CONNECT</a></h3>
```

Comment:
:red_circle: Please make sure you remove the underline applied on  this text. 
You can add this code to your style.css to override the default value.
```
a {
  text-decoration: none
} 
``` 
|Your Design|Figma Design|
|:--------------:|:------------------:|
|![your desi](https://user-images.githubusercontent.com/26736582/212391203-91c05aa8-bb54-4f61-a1c6-66b839d3241b.png)|![figmaa](https://user-images.githubusercontent.com/26736582/212391320-491b6d18-7913-41fd-adc3-5aebac95123b.png)|